### PR TITLE
Tickets/dm 32873

### PR DESCRIPTION
**Pull request by pkubanek**
Save hardpoint actuator data as well. Bypassing EFD, as in salobj it's pretty trivial to add handlers listening to data, and that's EFD independent - there are other tests designed to test EFD, not this one. I might re-introduce EFD check later. Added script to plot data, enabling quick check that HP are behaving as expected.

### CODE REVIEW

File: M13T004.py [hunk 1]
```diff
@@ -49,23 +49,56 @@
...
 
-class M13T004(MTM1M3Test):
+class AutoFlush:
+    """Automatic file flusher.
+
+    Parameters
+    ----------
+    file : `file`
+        File where message will be printed.
+    flushAfter : `int`
```
Comment (teweitsai) [inline M13T004.py:59]: This is "optional".

File: PlotT004.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+
+# This file is part of ts_salobj.
```
Comment (teweitsai) [inline PlotT004.py:3]: `ts_salobj` or `M1M3 test suite` in `M13T004.py`?

File: PlotT004.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+maxcols = math.ceil(len(args.files) / rowscols)
+
+print(f"Subplots: {rowscols}, {maxcols}")
+fig, axes = plt.subplots(rowscols, maxcols)
+
+count = 0
+
+
+def plot(fn):
+    global count, args
```
Comment (teweitsai) [inline PlotT004.py:54]: Do you need this `global` here?

**Review by teweitsai — APPROVED** [review]
The PR looks good to me and useful.

File: PlotT004.py (hunk 3 above)
Comment (pkubanek) [reply PlotT004.py:54]: Probably yes. I will try to solve & document that in next PR.

**Pull request merged**